pkg/network/setup/netconf.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package network
+
+import (
+	"fmt"
+	"sync"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/kubevirt/pkg/network/cache"
+)
+
+type NetConf struct {
```

Comment:
It started as a "controller", but then it was obvious there are two parts here.. the configuration and the status/state.
I think "netconf" is a common name for network configuration, and you just make it a stronger case when giving the CNI as the example. It does represent the network configuration, and I find it a fit.
But if you have something else, please propose it. In general, if we can avoid "manager", "controller" and other generic naming, it will be nice, but I will not dig into this too much.